release.rst

```diff
@@ -0,0 +1,54 @@
+Instructions for Creating a DAGMC Release
+============================================
+
+Because of a complex dependency between DAGMC and PyNE the following procedures
+are recommended for navigating those dependencies for a new release.
+
+DAGMC depends on PyNE only as a submodule from which C++ files are directly
+incorporated into the build.  It may be desirable to perform a patch-level
+release of DAGMC when any of the C++ files on PyNE change.
+
+PyNE depends on DAGMC as a third-party library that must be installed prior to
+building PyNE.  Changes in DAGMC do not necessarily require updates to PyNE, 
+so may not result in a patch-level release of PyNE.
+
+It is often desirable to perform patch-level releases of PyNE and DAGMC on
+concert.  If so, the following procedure should reduce conflicts in the
+dependencies.
+
+Assumptions
+------------
+
+1. The DAGMC repository has a `stable` tag that points to the same SHA as the newest release.
+2. The PyNE CI docker images rely on the DAGMC `stable` tag (this was incorporated in PyNE PR #1415)
```

Comment:
```suggestion
2. The PyNE CI docker images rely on the DAGMC `stable` Docker image tag (this was incorporated in PyNE PR #1415)
```

Reply:
ibid

Reply:
We could add a [link](https://github.com/svalinn/DAGMC/pkgs/container/dagmc-ci-ubuntu-20.04) to the docker image is that helps